src/display-layer.coffee

```diff
@@ -126,6 +127,9 @@ class DisplayLayer
       "\r\n": @invisibles.cr + @invisibles.eol
     }
 
+  recomputeScreenLinesIfNeeded: ->
```

Comment:
What about renaming this to `recomputeAllSpatialScreenLinesIfNeeded()`? In this file we usually refer to "screen lines" when talking about the synthesized lines that contain both spatial information and decoration tags.